File: Indie Python Programming Course/step03/3.03/3.3.01.py
'''

В данной задаче необходимо сравнить два целых числа A и B. Они поступают на вход программе отдельно в каждой строке.

Ваша задача вывести символ <, если A меньше B, >, если A больше B и =, если A=B.

Sample Input 1:

5
9

Sample Output 1:

<

Sample Input 2:

2
1

Sample Output 2:

>

Sample Input 3:

7
7

Sample Output 3:

=

'''

a, b = int(input()), int(input())
if a > b:
    print('>')
elif a < b:
    print('<')
else:
    print('=')

File: JavaScript. A3 Tasks/3 JavaScript для начинающих/3.03/3.3.01.js
/*

Найдите сумму  всех целых чисел от 1 до n включительно и верните из функции результат.

Sample Input 1:

8

Sample Output 1:

36

Sample Input 2:

2

Sample Output 2:

3

*/

function testCycle(n) {
  let x = 0;
  for (let i = 0; i <= n; i++){
      x+=i;
  }
  return x;
}

let n = Number(prompt())
console.log(testCycle(n))
